src/ipc/ipc3/handler.c

```diff
@@ -461,7 +461,6 @@ int ipc_stream_drain_notify(void)
 
 	/* send IPC notification */
 	struct ipc_msg *msg = ipc_msg_init(drain.rhdr.hdr.cmd, drain.rhdr.hdr.size);
-
 	ipc_msg_send(msg, &drain, false);
```

Comment:
The removal of this new line shouldn't be here. It doesn't add any value to this patch. If you really want to remove it, just remove it at the initial patch.

Reply:
Fixed.